svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java

```diff
@@ -224,6 +225,10 @@ String createJwt() throws BipException {
     claims.put("iss", bipApiProps.getClaimIssuer());
     byte[] signSecretBytes = bipApiProps.getClaimSecret().getBytes(StandardCharsets.UTF_8);
     Key signingKey = new SecretKeySpec(signSecretBytes, SignatureAlgorithm.HS256.getJcaName());
+
+    String secretString = Encoders.BASE64.encode(signingKey.getEncoded());
+    System.out.println(bipApiProps.getClaimSecret());
```

Comment:
I would expect linters to complain about `System.out.println` calls. Why are these lines added?

Reply:
whoops, i was debugging something and left this in, will rm it